index.html

```diff
@@ -5,38 +5,183 @@
+            <li class="menu__item mark"><a href="#">Century Concert</a></li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <a class="hamburger-menu__about" href="#"><img src="./icons/menu.png" alt="hamburger menu"></a>
+    <div class="about-general">
+      <h3>"Welcome music lovers"</h3>
+      <h1 class="about__title">THE CONCERT OF THE CENTURY <br>THE MOST SPECTACULAR MEETING OF POP AND ROCK IN 2022</h1>
+      <p class="paragraph__about">As in the most important cities in Europe and the United States, the biggest international artists have set their attention this time on Colombia to stage their shows. Do not miss this great opportunity to witness the biggest Anglo rock party, with artists who are music legends.</p>
+      <small>Please contact us per email for any further question about <br>THE CONCERT OF THE CENTURY! <br> <span class="mail">concerts2022@allshows.com</span></small>
+    </div>
+    <div class="about-information">
+      <div class="about-information__title">The Concert of the Century</div>
+      <p>Our name and logo are inspired as tribute to the main instrument in the rock environment, beside million of people that love this musical art.</p>
+      <a class="about-information__logo" href="#"><img src="./images/logo-rock2.jpg" alt="header icon"></a>
+      <div class="about-information__title">See the last two Organized for Us</div>
+      <p>Take a look at the last two musical events which took place in Santiago de Chile and Toronto.</p>
+      <div class="concerts">
+        <div class="concert1">2019 <br>The best Concert in Chile</div>
+        <div class="concert2">2021<br>Full house at Toronto Stadium</div>
+      </div>
     </div>
-    <nav>
-      <a class="menu" href="#"><img src="./icons/hamburger.png" alt="hamburger menu"></a>
-      <ul class="menu__list">
-        <li class="menu__list-item"><a href="#projects">Portfolio</a></li>
-        <li class="menu__list-item"><a href="#about-me">About</a></li>
-        <li class="menu__list-item"><a href="#contact">Contact</a></li>
+    <footer>
+      <h3>Partner</h3>
+      <div class="underline"></div>
+      <ul class="sponsors">
+        <li><a href="#"><img src="./images/mozilla-logo.png" alt=""></a></li>
+        <li><a href="#"><img src="./images/google-logo.svg" alt=""></a></li>
+        <li><a href="#"><img src="./images/naver-logo.png" alt=""></a></li>
+        <li><a href="#"><img src="./images/daumkakao-logo.png" alt=""></a></li>
+        <li><img src="./images/airbnb-logo.svg" alt=""></li>
       </ul>
-    </nav>
-  </header>
-  <main>
-  </main>
+    </footer>
+    <ul class="last-section">
+      <li>
+        <img src="./images/logo-rock2.jpg" alt="">
+      </li>
+      <li>
+        <p>All content on this website, unless otherwise specified, is available under a Creative Commons Atrribution<br>Copyright © 2022. <br>All rights reserved.</p>
+      </li>
+    </ul>
+  </section>
```

Comment:
Hello Hammas. I did not find your suggestion as a Microverse requirement at least in this project. If you could check the functionality is okay, it is working and agrees with the requirements. However, you have a reason for the future projects, but in this case, I did not find that was forbidden work in a single-page application. In fact, it is the first time I hear about it because I do not have the criteria to choose a 'single' or 'double' page application. Surely, I will take this into account for the next projects